### DISCUSSION TITLE
Different debugging logs

### DISCUSSION THREAD

**Discussion by vajrang-b** [vajrang-b/RPA-Developer-in-30-Days#19] [Q&A] [answered]
logs
Write line
Messagebox

**Comment by Manjushree15** [comment, marked as answer]
1. Logs: Logging level of the message can be selected from the dropdown list(Different log levels: info, error, warn, Trace)
2. Write Line: Logs the message in the output panel with trace log level.
3. Message Box: Displays message box with specified text.

**Comment by TiwariRakesh** [comment]
Logs- useful in monitoring the execution of BOT(it has different logging)
Writeline- prints the output in output panel of uipath studio.
Messagebox - another way of checking the expected output.

**Comment by Ulhasguttikar** [comment]
Logs :- In Log Message we can specify the severity of the Message by selecting the options in level drop down. Writes the specified diagnostic message at the specified level. These messages are also sent to Orchestrator and displayed in the Logs page.
Write line :- Write line is a Log Message with Trace Level severity.
Messagebox :- Displays a message box with the specified text and button options.
